draft-ietf-rats-corim.md

```diff
@@ -914,8 +914,8 @@ $version-scheme /= int / text
 
 The following details the security version number (`svn`) and the minimum security version number (`min-svn`) statements.
 A security version number is used to track changes to an object (e.g., a secure enclave, a boot loader executable, a configuration file, etc.) that are security relevant.
-Rollback of a security relevant change is considered to be an attack vector, as such, security version numbers can't be decremented.
-If a security relevant flaw is discovered in the Target Environment and subsequently fiexed, the `svn` value is typically incremented.
+Rollback of a security relevant change is considered to be an attack vector and as such, security version numbers cannot be decremented.
```

Comment:
```suggestion
Rollback of a security relevant change is considered to be an attack vector; as such, security version numbers MUST NOT be decremented.
```

Reply:
I updated the editorial suggestions, but I do not want to add or remove normative wording in this pull request.